src/collectors/ebpf.plugin/ebpf_shm.h

```diff
@@ -23,10 +23,10 @@
 #define NETDATA_CGROUP_SHM_DT_CONTEXT "cgroup.shmdt"
 #define NETDATA_CGROUP_SHM_CTL_CONTEXT "cgroup.shmctl"
 
-#define NETDATA_SYSTEMD_SHM_GET_CONTEXT "services.shmget"
-#define NETDATA_SYSTEMD_SHM_AT_CONTEXT "services.shmat"
-#define NETDATA_SYSTEMD_SHM_DT_CONTEXT "services.shmdt"
-#define NETDATA_SYSTEMD_SHM_CTL_CONTEXT "services.shmctl"
+#define NETDATA_SYSTEMD_SHM_GET_CONTEXT "systemd.services.shmget"
```

Comment:
Should all this use NETDATA_SERVICE_FAMILY instead of `systemd` here?

Reply:
This is the pattern used for contexts in our `cgroups.plugin`: 

```sh
"systemd_systemd-networkd.mem_usage":{
            "id":"systemd_systemd-networkd.mem_usage",
            "name":"systemd_systemd-networkd.mem_usage",
            "type":"systemd_systemd-networkd",
            "family":"mem",
            "context":"systemd.service.memory.usage",
            "title":"Systemd Services Used Memory (systemd_systemd-networkd.mem_usage)",
            "priority":19005,
            "plugin":"cgroups.plugin",
```

This is the reason, we are keeping the same pattern here.